### DISCUSSION TITLE
[Github Projects] Filter Out Issues From Closed Milestones

### DISCUSSION THREAD

**Discussion by dflor003** [community/community#27526] [Projects and Issues]
Just started using the new Github Projects (which looks like it is no longer in Beta). We migrated our existing "classic" Github project and it looks like all of the issues for closed milestones are showing up in the table view. We would really like to be able to filter those out somehow. Perhaps something like this?

```
is:milestone is:open
```

**Comment by leandro-lucarella-frequenz** [comment]
This is particularly useful when having a view grouped by milestone, which makes for a really nice roadmap view. In that case showing closed milestones adds a lot of noise (that normally will also show up first, which makes it even more noisy). The only workaround for now is to manually filter all closed milestones, which is cumbersome.
